The new model is not found in opencv-extra:
```
[ RUN      ] DNNTestNetwork.YuNet_320/0, where GetParam() = OCV/CPU
  TEST ERROR: Don't use 'optional' findData() for dnn/onnx/models/yunet-202605.onnx
  /home/ci/opencv/modules/ts/src/ts_perf.cpp:1968: Failure
  Failed
  Expected: PerfTestBody() doesn't throw an exception.
    Actual: it throws cv::Exception:
    OpenCV(5.0.0-pre) /home/ci/opencv/modules/ts/src/ts.cpp:969: error: (-215:Assertion failed) required || result_.empty() in function 'findData'
  
  params    =     OCV/CPU
  termination reason:  unhandled exception
  bytesIn   =          0
  bytesOut  =          0
  samples   =          0 of 1
  outliers  =          0
  frequency =          0
  [  FAILED  ] DNNTestNetwork.YuNet_320/0, where GetParam() = OCV/CPU (1 ms)
  [ RUN      ] DNNTestNetwork.YuNet_640/0, where GetParam() = OCV/CPU
  TEST ERROR: Don't use 'optional' findData() for dnn/onnx/models/yunet-202605.onnx
  /home/ci/opencv/modules/ts/src/ts_perf.cpp:1968: Failure
  Failed
  Expected: PerfTestBody() doesn't throw an exception.
    Actual: it throws cv::Exception:
    OpenCV(5.0.0-pre) /home/ci/opencv/modules/ts/src/ts.cpp:969: error: (-215:Assertion failed) required || result_.empty() in function 'findData'
  
  params    =     OCV/CPU
  termination reason:  unhandled exception
  bytesIn   =          0
  bytesOut  =          0
  samples   =          0 of 1
  outliers  =          0
  frequency =          0
  [  FAILED  ] DNNTestNetwork.YuNet_640/0, where GetParam() = OCV/CPU (2 ms)
  [ RUN      ] DNNTestNetwork.YuNet_1280/0, where GetParam() = OCV/CPU
  TEST ERROR: Don't use 'optional' findData() for dnn/onnx/models/yunet-202605.onnx
  /home/ci/opencv/modules/ts/src/ts_perf.cpp:1968: Failure
  Failed
  Expected: PerfTestBody() doesn't throw an exception.
    Actual: it throws cv::Exception:
    OpenCV(5.0.0-pre) /home/ci/opencv/modules/ts/src/ts.cpp:969: error: (-215:Assertion failed) required || result_.empty() in function 'findData'
  
  params    =     OCV/CPU
  termination reason:  unhandled exception
  bytesIn   =          0
  bytesOut  =          0
  samples   =          0 of 1
  outliers  =          0
  frequency =          0
  [  FAILED  ] DNNTestNetwork.YuNet_1280/0, where GetParam() = OCV/CPU (6 ms)
  [----------] Global test environment tear-down
  [ SKIPSTAT ] 11 tests skipped
  [ SKIPSTAT ] TAG='verylong' skip 11 tests
  [==========] 1100 tests from 27 test cases ran. (17781 ms total)
  [  PASSED  ] 1097 tests.
  [  FAILED  ] 3 tests, listed below:
  [  FAILED  ] DNNTestNetwork.YuNet_320/0, where GetParam() = OCV/CPU
  [  FAILED  ] DNNTestNetwork.YuNet_640/0, where GetParam() = OCV/CPU
  [  FAILED  ] DNNTestNetwork.YuNet_1280/0, where GetParam() = OCV/CPU
```

--- a/modules/dnn/perf/perf_net.cpp
+++ b/modules/dnn/perf/perf_net.cpp
@@ -30,7 +30,7 @@ public:
 
     void processNet(std::string weights, std::string proto,
                     const std::vector<std::tuple<Mat, std::string>>& inputs, const std::string& outputLayer = ""){
-        weights = findDataFile(weights, false);
+        weights = findDataFile(weights);
         if (!proto.empty())
             proto = findDataFile(proto);
         net = readNet(weights, proto);

--- a/modules/dnn/test/test_backends.cpp
+++ b/modules/dnn/test/test_backends.cpp
@@ -395,13 +395,10 @@ TEST_P(DNNTestNetwork, OpenPose_pose_mpi_faster_4_stages)
 TEST_P(DNNTestNetwork, YuNet)
 {
     Mat img = imread(findDataFile("gpu/lbpcascade/er.png"));
-    for (const auto& sz : {Size(320, 320), Size(640, 480), Size(640, 640)})
-    {
-        Mat resized;
-        resize(img, resized, sz);
-        Mat inp = blobFromImage(resized);
-        processNet("dnn/onnx/models/yunet-202605.onnx", "", inp);
-    }
+    Mat resized;
+    resize(img, resized, Size(320, 320));
+    Mat inp = blobFromImage(resized);
+    processNet("dnn/onnx/models/yunet-202605.onnx", "", inp);
     expectNoFallbacksFromIE(net);
 }
 

--- a/modules/dnn/test/test_backends.cpp
+++ b/modules/dnn/test/test_backends.cpp
@@ -33,7 +33,7 @@ public:
         l1 = l1 ? l1 : default_l1;
         lInf = lInf ? lInf : default_lInf;
 
-        weights = findDataFile(weights, false);
+        weights = findDataFile(weights);
         if (!proto.empty())
             proto = findDataFile(proto);
 

